.gitignore

```diff
@@ -9,3 +9,4 @@ log/*
 .envrc
 .tool-versions
 newrelic_agent.log
+.vscode
```

Comment:
This is minor (and ignorable). I've worked with teams who prefer not to have editor files as part of .gitignore, and instead for each person using the editor to add these entries to `.git/info/exclude` (a local-only .gitignore file).

I don't have a strong preference one way or another, but wanted to call it out in case that's a similar path we want to take.

Reply:
Hmm, yeah. Maybe we need a wider team discussion on that. `slack` has `.vscode` in the `.gitignore`. Generally I feel like editor configurations are very user-specific. That's partially because I end up running the app differently than the rest of the team on both slack and jira haha. I can leave it in and locally ignore it, but I significantly changed the `launch.json` file for my own needs and I had to hard-code the path to my local `node` install (still haven't figured out why yet) so that was my motivation for removing. LMK if you want me to add it back.

Reply:
I think this is all good for now. Yeah let's bubble this up to a team discussion to see if there's consensus one direction or another.

Honestly it doesn't seem like there's a ton of negative consequences one way or another (at least that I'm seeing, but I could be missing something).

Reply:
There's also the system level excludes file. IDE (and OS) -specific ignores will likely be identical for a given user on a given machine, so adding it to the system level ignore file means you don't need to add it manually to `.git/info/exclude` in every repo on your system.

I'm also a pretty staunch supporter of minimal project gitignores in order to elevate the signal-to-noise ratio.

Reply:
I think that's a reasonable stance!